app/views/find_local_council/district_and_county_council.html.erb

```diff
@@ -27,11 +27,11 @@
         <% end %>
       </p>
 
-      <p class="govuk-body">County councils are responsible for services like:</p>
+      <p class="govuk-body"><%= t('formats.local_transaction.county_council_services')%>:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>education</li>
-        <li>transport</li>
-        <li>social care</li>
+        <li><%= t('education') %></li>
```

Comment:
A nitpicky comment: could this be listed in the locale files as an array then just loop through each item here?  It would allow the list to be updated by simply changing the locale files.